README.md

```diff
@@ -17,17 +17,66 @@ The Planet SDK for Python is in 'pre-release' stages, working towards an
 initial release around July. Active development is tracked in the [Planet SDK 
 for Python Project](https://github.com/planetlabs/planet-client-python/projects/2). 
 The initial release will support Orders, Data and Subscription API's in the 
-command-line interface, with corresponding Python libraries. We expect 'beta' 
-milestones to be released in some form for each of the API's. After the 
+command-line interface, with corresponding Python libraries. After the 
 initial July release there will be additional work to support the remaining 
 Planet API's ([analytics](https://developers.planet.com/docs/analytics/), 
 [basemaps](https://developers.planet.com/docs/basemaps/) and 
 [tasking](https://developers.planet.com/docs/tasking/)).
 
+## Installation
+
+There are two options for installation: installing from PyPi or installing from source.
+
+Be aware that using either route will by default update your `planet` command-line and
+python library to v2, so you will no longer be able to use v1. As v2 is in a pre-release
+stage, with not all functionality implemented, we strongly recommend installing it in a 
+virtual environment - either [venv](https://python.land/virtual-environments/virtualenv) 
+or [conda](https://docs.conda.io/projects/conda/en/latest/user-guide/getting-started.html). 
+If you've never worked with either then check out our [getting started with venv and Planet 
+SDK](docs/venv-tutorial.md).
```

Comment:
It may be worth including a line that says something like "For those planning to contribute to the SDK, see the contributing guide(link to CONTRIBUTING.md) for vitualenv suggestions" - this is because the contributing guide suggests using pyenv.